lib/pages/dashboard/blocs/team_classification_bloc.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:v34/models/classication.dart';
+import 'package:v34/models/team.dart';
+import 'package:v34/repositories/repository.dart';
+
+// -- Events
+
+class TeamClassificationEvent extends Equatable {
+  final Team team;
+
+  const TeamClassificationEvent(this.team) : assert(team != null);
+
+  @override
+  List<Object> get props => [team];
+}
+
+class TeamClassificationLoadEvent extends TeamClassificationEvent {
+  TeamClassificationLoadEvent(Team team) : super(team);
+}
+
+// -- States
+
+class TeamClassificationState extends Equatable {
+  final String highlightedTeamCode;
+  final List<ClassificationSynthesis> classifications;
+
+  TeamClassificationState(this.highlightedTeamCode, this.classifications);
+
+  @override
+  List<Object> get props => [highlightedTeamCode, classifications];
+
+  @override
+  String toString() {
+    return "TeamClassification(highlightedTeamCode: $highlightedTeamCode, classifications: $classifications)";
+  }
+}
+
+class TeamClassificationUninitializedState extends TeamClassificationState {
+  TeamClassificationUninitializedState(String highlightedTeamCode, List<ClassificationSynthesis> classifications)
```

Comment:
Because this is an uninitialized state, you don't need those parameters in the constructor.
```suggestion
  TeamClassificationUninitializedState(): super(null, [])
```